src/test/java/com/artipie/FilesRepoITCase.java

```diff
@@ -168,9 +154,20 @@ void tearDown() {
         this.cntn.stop();
     }
 
-    private String exec(final String... command) throws Exception {
-        Logger.debug(this, "Command:\n%s", String.join(" ", command));
-        return this.cntn.execInContainer(command).getStdout();
+    private String curl(final boolean anonymous, final String action,
+        final ArtipieServer.User usr) throws Exception {
+        final String url = "http://host.testcontainers.internal:%d/my-file/file-repo/curl.txt";
+        final List<String> cmdlst = new ArrayList<>(
+            Arrays.asList(
+                "curl", "-i", "-X", action, String.format(url, this.port)
+            )
+        );
+        cmdlst.add(this.flag(anonymous));
+        cmdlst.add(this.user(anonymous, usr));
+        String[] cmdarr = new String[cmdlst.size()];
+        cmdarr = cmdlst.toArray(cmdarr);
```

Comment:
@genryxy why not just change these two lines into `String[] cmdarr = cmdlst.toArray(new String[cmdlst.size()]);`, first assignment is just not needed and is mind-boggling